docs/content-services/content-metadata.component.md

```diff
@@ -27,6 +27,8 @@ The different aspects and their properties to be shown can be configured as appl
 | --- | --- | --- | --- |
 | node | MinimalNodeEntryEntity | - | (**required**) The node entity to fetch metadata about |
 | displayEmpty | boolean | false | Display empty values in card view or not |
+| canEdit | boolean | true | Whether the edit button to be shown or not |
```

Comment:
can we change the name of this parameter in readOnly as for the other component?